manifoldco/ui
# Use ownerId in manifold-data-resource-list and manifold-resource-list

## Reason for change

In both `<manifold-data-resource-list>` and `<manifold-resource-list>`:
- add ownerId prop
- include owner argument to `resources` graphql query

## Testing

Per @domenicrosati, we don't have a way to create resources with anything other than our user's ownerId, so unfortunately we can't fully demonstrate filtered lists that look any different than a user's full list of resources. Until we can do that, the acceptance criteria is:

1. Make sure both components work without an ownerId
2. Make sure they work the same with a valid ownerId
3. Make sure they fail on an invalid ownerId

I've added an ownerId knob to both resource -> list and resource -> list (unstyled) stories to test this.

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
